modules/ROOT/pages/functions/faq.adoc

```diff
@@ -0,0 +1,258 @@
+Serverless Functions FAQ
+========================
```

Comment:
Please use heading titles as per https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#section-titles
So this should be a = H1, subsequent headings should be == H2 etc